src/profile/supporting_types/detailed_authorized_personas.rs

```diff
@@ -0,0 +1,38 @@
+use crate::prelude::*;
+
+decl_can_be_empty_identified_array_of!(
+    DetailedAuthorizedPersonas,
```

Comment:
unsure about the plural name here, but `AuthorizedPersonasDetailed` does not show that it is plural as clearly. `AuthorizedDetailedPersonas` perhaps? If so we ought to renamed `AuthorizedPersonaDetailed` -> `AuthorizedDetailedPersona`, that works. And probably renamed `AuthorizedPersonaSimple` -> `AuthorizedSimplePersona`? Hmm but it sounds like the user has "authorized a `SimplePersona`". Which is not true, instead the user has "authorized a Persona - and this is the most distilled / stripped representation. hmm.